dask/dataframe/methods.py

```diff
@@ -480,7 +482,68 @@ def concat_pandas(dfs, axis=0, join="outer", uniform=False, filter_warning=True)
     return out
 
 
+# ---------------------------------
+# hash_df
+# ---------------------------------
+
+
+hash_df = Dispatch("hash_df")
+
+
+@hash_df.register((pd.DataFrame, pd.Series, pd.Index))
+def hash_df_pandas(dfs):
+    return hash_pandas_object(dfs, index=False)
+
+
+# ---------------------------------
+# group_split
+# ---------------------------------
+
+
+group_split = Dispatch("group_split")
+
+
+@group_split.register((pd.DataFrame, pd.Series, pd.Index))
+def group_split_pandas(df, c, k):
+    indexer, locations = groupsort_indexer(c.astype(np.int64), k)
+    df2 = df.take(indexer)
+    locations = locations.cumsum()
+    parts = [df2.iloc[a:b] for a, b in zip(locations[:-1], locations[1:])]
+
+    return dict(zip(range(k), parts))
+
+
+# ---------------------------------
+# group_split_2
+# ---------------------------------
```

Comment:
We probably don't need separate comments like these for each function.  Maybe one larger header for "shuffling/merging/sorting" or something like that?